app/controllers/tattoos_controller.rb

```diff
@@ -71,6 +71,16 @@ def destroy
     end
   end
 
+  def best
+    tattoos = Tattoo.all
+    best_tattoos = []
+    tattoos.each do |tattoo|
+      best_tattoos << tattoo
+    end
```

Comment:
^ This part doesn't make much sense to me - aren't you just taking all the tattoos and putting them in an array? Can you not all `.sort!` or `.to_a` on an active record relation?